core/station/impl/src/services/system.rs

```diff
@@ -626,4 +628,44 @@ mod tests {
 
         assert!(system_info.get_change_canister_request().is_none());
     }
+
+    #[test]
+    fn test_initial_quorum_is_majority() {
+        assert_eq!(calc_initial_quorum(1, None), 1);
+        assert_eq!(calc_initial_quorum(2, None), 2);
+        assert_eq!(calc_initial_quorum(3, None), 2);
+        assert_eq!(calc_initial_quorum(4, None), 3);
+        assert_eq!(calc_initial_quorum(5, None), 3);
+        assert_eq!(calc_initial_quorum(6, None), 4);
+        assert_eq!(calc_initial_quorum(7, None), 4);
+        assert_eq!(calc_initial_quorum(8, None), 5);
+        assert_eq!(calc_initial_quorum(9, None), 5);
+        assert_eq!(calc_initial_quorum(10, None), 6);
+        assert_eq!(calc_initial_quorum(11, None), 6);
+        assert_eq!(calc_initial_quorum(12, None), 7);
+        assert_eq!(calc_initial_quorum(13, None), 7);
+        assert_eq!(calc_initial_quorum(14, None), 8);
+        assert_eq!(calc_initial_quorum(15, None), 8);
+        assert_eq!(calc_initial_quorum(16, None), 9);
+    }
+
+    #[test]
+    fn test_initial_quorum_is_custom() {
```

Comment:
Could we have tests where Some(n) is
- 1
- half of the number of admins
- equal to the number of admins
- larger than the number of admins

Reply:
This would be more useful than a lot of tests with Some(n) where n is always half of the number of admins.